features/git-new-pull-request/github.feature

```diff
@@ -27,3 +29,29 @@ Feature: git-new-pull-request when origin is on GitHub
       | https://github.com/Originate/originate.github.com     | Originate/originate.github.com |
       | git@github.com:Originate/originate.github.com.git     | Originate/originate.github.com |
       | git@github.com:Originate/originate.github.com         | Originate/originate.github.com |
+
+
+  Scenario: nested feature branch with known parent
+    Given I have a feature branch named "parent-feature"
+    And I have a feature branch named "child-feature" as a child of "parent-feature"
+    And my remote origin is git@github.com:Originate/git-town.git
+    And I am on the "child-feature" branch
+    When I run `git new-pull-request`
+    Then I see a new GitHub pull request for the "child-feature" branch against the "parent-feature" branch in the "Originate/git-town" repo in my browser
+
+
+  Scenario: nested feature branch with unknown parent (entering the parent name)
+    Given I have a feature branch named "feature"
+    And Git Town has no branch hierarchy information for "feature"
+    And my remote origin is git@github.com:Originate/git-town.git
+    And I am on the "feature" branch
+    When I run `git new-pull-request` and enter "main"
+    Then I see a new GitHub pull request for the "feature" branch against the "main" branch in the "Originate/git-town" repo in my browser
+
+  Scenario: nested feature branch with unknown parent (accepting default choice)
+    Given I have a feature branch named "feature"
+    And Git Town has no branch hierarchy information for "feature"
+    And my remote origin is git@github.com:Originate/git-town.git
+    And I am on the "feature" branch
+    When I run `git new-pull-request` and enter ""
```

Comment:
how about making this step more readable by just saying `and press enter` instead of `and enter ""`